apps/docs/src/guide/predicates/send-and-spend-funds-from-predicates.md

```diff
@@ -38,6 +38,14 @@ Note the method transfer has two parameters: the recipient's address and the int
 
 Once the predicate resolves with a return value `true` based on its predefined condition, our predicate successfully spends its funds by means of a transfer to a desired wallet.
 
+---
+
+In a similar approach, you can use the `createTransfer` method, which returns a `ScriptTransactionRequest`. Then, we can submit this transaction request by calling the `sendTransaction` method.
+
+This can be useful if you need the transaction ID before actually submitting it to the node.
```

Comment:
Wondering whether we need some docs here or _somewhere_ around how tx ids are formed, and the risk of it changing if the request is changed?